modules/gentemplates/openapi-generator/src/main/java/com/reprezen/genflow/openapi/generator/OagCodegenGenTemplateBase.java

```diff
@@ -64,6 +64,32 @@ public void configure() throws GenerationException {
 				"Each property should be a JSON object with a name/value pair for each property.",
 				"Example: for '-Dmodels -Dapis=User,Pets' use the following:", "value:", "  models: ''",
 				"  apis: Users,Pets"));
+		define(parameter().named(LANGUAGE_SPECIFIC_PRIMITIVES).optional().withDescription(
+				"Specifies additional language specific primitive types.",
+				"You can have multiple occurrences of this option.", //
+				"Example: -Pet", //
+				"         -User"));
```

Comment:
The example is a bit confusing because it looks like the "Example:" is part of the YAML content. Also, is it really "multiple _occurrences of this option_" or is one occurrence with _multiple values_ provided as array elements? Given your example, I am assuming the latter.

My suggestion is to replace the last three lines with these four lines:

> The value is an array of type names. For example: 
>  LANGUAGE_SPECIFIC_PRIMITIVES:
>  - Pet
>  - User